lisa/base_tools/wget.py

```diff
@@ -21,13 +37,21 @@ def can_install(self) -> bool:
         return True
 
     def install(self) -> bool:
-        posix_os: Posix = cast(Posix, self.node.os)
-        posix_os.install_packages([self])
+        if not self._check_exists():
```

Comment:
This line is not necessary. The check will be managed by LISA itself.

Reply:
Done

Reply:
I don't get why you're moving away from casting here.

Reply:
hmm weird, it doesn't show me in local changes and also in files view. 
I think you might be looking at an older iteration.